scripts/devops_tasks/tox_harness.py

```diff
@@ -274,11 +274,66 @@ def replace_dev_reqs(file, pkg_root):
         f.write("\n".join(adjusted_req_lines))
 
 
+def collect_log_files(working_dir):
+    logging.info("Collecting log files from {}".format(working_dir))
+    package = working_dir.split('/')[-1]
+    # collect all the log files into one place for publishing in case of tox failure
+    test_envs = ["whl", "sdist", "depends", "latestdependency", "mindependency"]
```

Comment:
I kinda feel like we should do a surface level glob of the folders under the `.tox` folder, excluding `.tmp`. `glob.glob` totally works on py27 as long as we don't need recursion. In this case, we do not.

That way, if we ever add another environment, we won't have to remember to update this list.